src/h3/unstable/v4.py

```diff
@@ -0,0 +1,135 @@
+# flake8: noqa
+
+# todo: what are the pythonic names?
+
+# what if we put this in an object? what would it look like?
+# a temporary API wrapper
+
+# what if we organized by module
+
+from ..api.basic_str import (
+    compact as
+        compact,
+
+    edge_length as
+        hex_edge_length_avg,
+
+    geo_to_h3 as
+        point_to_cell,
+
+    get_destination_h3_index_from_unidirectional_edge as
+        edge_destination,
+
+    get_h3_indexes_from_unidirectional_edge as
+        directed_edge_cells,
+
+    get_h3_unidirectional_edge as
+        cells_to_directed_edge,
+
+    get_h3_unidirectional_edge_boundary as
+        directed_edge_boundary, # what's a boundary?
+
+    get_h3_unidirectional_edges_from_hexagon as
+        origin_to_directed_edges,
+
+    get_origin_h3_index_from_unidirectional_edge as
+        edge_origin,
+
+    get_pentagon_indexes as
+        pentagons,
+
+    get_res0_indexes as
+        res0_cells,
+
+    h3_distance as
+        grid_distance,
+
+    h3_get_base_cell as
+        base_cell_number,
+
+    h3_get_faces as
+        faces,
```

Comment:
This one does seem a little obscure as a function name rather than a method on an index object.

Reply:
Agreed. Going with a more verbose name for faces.